dashboard/app/label.go

```diff
@@ -227,6 +227,25 @@ func (bug *Bug) UnsetLabels(labels ...BugLabelType) map[BugLabelType]struct{} {
 	return notFound
 }
 
+func (bug *Bug) UnsetLabelValues(values ...string) map[string]struct{} {
+	toDelete := map[string]struct{}{}
+	notFound := map[string]struct{}{}
+	for _, val := range values {
+		toDelete[val] = struct{}{}
+		notFound[val] = struct{}{}
+	}
+	var newList []BugLabel
+	for _, item := range bug.Labels {
+		if _, ok := toDelete[item.Value]; ok && item.Value != "" {
```

Comment:
`if slices.Contains(values, item.Value) && item.Value != "" {}` ?
It removes 1 local variable.